e2e/jsonschema/match.go

```diff
@@ -66,7 +66,12 @@ func MatchEntities(data []*sdk.EntityData, schemaFileByJobByType map[string]Even
 	}
 
 	for _, entityData := range data {
-		metric := mergeMetricSets(entityData.Metrics)
+		// Merge metricSets from the entity
+		metric, err := mergeMetricSets(entityData.Metrics)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("merging metricSets for %s: %w", entityData.Metadata.Name, err))
```

Comment:
This allows to clearly identify the subject.
```suggestion
			errs = append(errs, fmt.Errorf("merging metricSets for %q: %w", entityData.Metadata.Name, err))
```